### PR TITLE
Bug 1845247 - Updated the ‘Discover more’ Pocket Story background to match the wallpaper colour

### PR DESCRIPTION
**Pull request by t-p-white**
wp4

wp1

wp2

wp3

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1845247

### CODE REVIEW

**Comment by t-p-white** [comment]
If needed, for info on how to build locally give me a ping on Slack

**Review by MatthewTighe — APPROVED** [review]
LGTM, thanks for catching and fixing this!

File: fenix/app/src/main/java/org/mozilla/fenix/compose/ListItemTabLargePlaceholder.kt [hunk 1]
```diff
@@ -37,19 +38,21 @@ import org.mozilla.fenix.theme.FirefoxTheme
  * ```
  *
  * @param text The only [String] that this will display.
+ * @param backgroundColor The background [Color] of the story.
```
Comment (gabrielluong) [inline fenix/app/src/main/java/org/mozilla/fenix/compose/ListItemTabLargePlaceholder.kt:41]:
```suggestion
 * @param backgroundColor The background [Color] of the list item placeholder.
```
nit: This is suppose to be a generic component.

File: fenix/app/src/main/java/org/mozilla/fenix/compose/ListItemTabLargePlaceholder.kt [hunk 2]
```diff
@@ -74,6 +77,9 @@ fun ListItemTabLargePlaceholder(
 @Preview
 private fun ListItemTabLargePlaceholderPreview() {
     FirefoxTheme {
-        ListItemTabLargePlaceholder(text = "Item placeholder")
+        ListItemTabLargePlaceholder(
+            text = "Item placeholder",
+            backgroundColor = FirefoxTheme.colors.layer2,
```
[the remark this answers is not in the document]
Comment (t-p-white) [reply fenix/app/src/main/java/org/mozilla/fenix/compose/ListItemTabLargePlaceholder.kt:82]: Thought about this 👍 as the parent call has the default set, I've kept it consistent with `PocketStory` & `PocketSponsoredStory`

**Pull request merged**